README.md

```diff
@@ -109,6 +109,32 @@ require("gitlab").setup({
     resolved = '✓', -- Symbol to show next to resolved discussions
     unresolved = '✖', -- Symbol to show next to unresolved discussions
   },
+  discussion_sign = {
+    -- See :h sign_define for details about sign configuration.
+    enabled = true,
+    text = "💬",
+    linehl = nil,
+    texthl = nil,
+    culhl = nil,
+    numhl = nil,
+    priority = 20,
+    helper_signs = {
+      -- For multiline comments the helper signs are used to indicate the whole context
+      -- Priority of helper signs is lower than the main sign (-1).
+      enabled = true,
+      start = "↑",
+      mid = "|",
+      ["end"] = "↓",
```

Comment:
This can actually happen when you create visual selection and press `o` to jump to other end of the selection you can actually create comment which will have sign at start and you would see end arrow like this 
![image](https://github.com/harrisoncramer/gitlab.nvim/assets/15018762/91183fbe-ab68-490c-96c7-c72d41c3afb0)